src/library/browse/foldertreemodel.cpp

```diff
@@ -59,7 +58,7 @@ bool FolderTreeModel::directoryHasChildren(const QString& path) const {
     }
 
     // Acquire a security token for the path.
-    MDir dir(path);
+    const auto dirAccess = mixxx::FileAccess(mixxx::FileInfo(path));
```

Comment:
This was intentional, because using mixxx::FileInfo is type-safe. The conversion from a generic string should only happen once when creating mixxx::FileInfo to spot and trace these conversions more easily. Clients should prefer to use mixxx::FileInfo instead of QString if the content represents a file system path. This is currently not always the case, i.e. often a meaningless QString is used.

Reply:
Ok